docs/config_search.md

```diff
@@ -303,6 +304,35 @@ After Model Analyzer has found the best config(s), it will then sweep the top-N
 
 ---
 
+## LLM Search
+
+_This mode has the following limitations:_
+
+- Summary/Detailed reports do not include the new metrics
+
+LLMs can be optimized using either Quick or Brute search mode by setting `--model-type LLM`. You can specify CLI options to the GenAI-Perf tool using `genai_perf_flags`. See the [GenAI-Perf CLI](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/genai-perf/README.md#cli) documentation for a list of the flags that can be specified.
+
+_An example model analyzer YAML config for a LLM:_
+
+```yaml
+model_repository: /path/to/model/repository/
+
+model_type: LLM
+client_prototcol: grpc
+
+genai_perf_flags:
+  backend: vllm
```

Comment:
The current recommended install path for GenAI-Perf installs 24.03, which names this `--output-format`

Reply:
Is this documentation to cater towards main branch of GenAI-Perf, or some specific release?

Reply:
This is documentation for 24.04